packages/insomnia-app/app/ui/components/settings/plugins.js

```diff
@@ -62,14 +64,16 @@ class Plugins extends React.PureComponent<Props, State> {
 
     const newState: $Shape<State> = {
       isInstallingFromNpm: false,
-      error: '',
+      error: null,
+      installPluginErrMsg: '',
     };
     try {
       await installPlugin(this.state.npmPluginValue.trim());
       await this._handleRefreshPlugins();
       newState.npmPluginValue = ''; // Clear input if successful install
     } catch (err) {
-      newState.error = err.message;
+      newState.installPluginErrMsg = `Failed to install ${this.state.npmPluginValue}`;
+      newState.error = err;
```

Comment:
We should only need to store `error` on state, and can move the text "Failed to install..." directly into the `render()` function

```suggestion
      newState.error = err;
```

Reply:
Okay! I originally added this new state because when using `npmPluginValue` in `render()` when the `error` is displayed and user types the text changing is reflected in the error.

Reply:
Oh I see! Makes sense. In that case this is fine. 🤗